Riva0110/Vocab-Garden
# Feature/plants phase

### Description

* Add plant-related logic at different score phase

### Test Plan

* Change score to check if the plant phase is correct
* Check if starting a challenge works successfully
